data.sql

```diff
@@ -68,4 +68,45 @@ INSERT INTO animals (
 ('Angemon', '2005-06-12', 1, TRUE, -45),
 ('Boarmon', '2005-06-07', 7, TRUE, 20.4),
 ('Blossom', '1998-10-13', 3, TRUE, 17),
-('Ditto', '2022-05-14', 4, TRUE, 22);
\ No newline at end of file
+('Ditto', '2022-05-14', 4, TRUE, 22);
+
+--ADD data to species table 
+
+INSERT INTO species (
+    name
+) VALUES 
+( 'Pokemon'),
+('Digimon');
+
+--POPULATE OWNERS TABLE 
+INSERT INTO owners (
+    full_name,
+    age
+) VALUES
+('Sam Smith', 34),
+('Jennifer Orwell', 19),
+('Bob', 45),
+('Melody Pond', 77),
+('Dean Winchester', 14),
+('Jodey Whittaker', 38);
+
+-- Update species_id in table animals
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

Reply:
Thank you @HAMMAS-SALEEM for the recommendation. I will implement that for sure. :100: